help/tquery.md

```diff
@@ -0,0 +1,52 @@
+# Tquery
+
+### Typy danych i operatory filtrów
+
+| typ          | null(5) | = | in   | cmp | like | select | sort |
+|--------------|---------|---|------|-----|------|--------|------|
+| bool         | *       | * | .    | .   | .    | *      | +    |
+| date         | *       | * | *    | *   | .    | *      | +    |
```

Comment:
Date i datetime też mają like za darmo więc też je włącz. To akurat może być przydatne że można wpisać kawałek daty w pole szukania.